File: test/fixtures/ruby/post_empty.rb
require 'net/http'

uri = URI('http://localhost:28139')
req = Net::HTTP::Post.new(uri)
req.content_type = 'application/x-www-form-urlencoded'

req.body = ''

req_options = {
  use_ssl: uri.scheme == 'https'
}
res = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
  http.request(req)
end

File: test/fixtures/swift/post_empty.swift
import Foundation

let data = NSMutableData(data: "".data(using: .utf8)!)

let url = URL(string: "http://localhost:28139")!
let headers = [
    "Content-Type": "application/x-www-form-urlencoded"
]

var request = URLRequest(url: url)
request.httpMethod = "POST"
request.allHTTPHeaderFields = headers
request.httpBody = data as Data

let task = URLSession.shared.dataTask(with: request) { (data, response, error) in
    if let error = error {
        print(error)
    } else if let data = data {
        let str = String(data: data, encoding: .utf8)
        print(str ?? "")
    }
}

task.resume()
